src/block/tile/Jukebox.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\block\tile;
+
+use pocketmine\world\World;
+use pocketmine\item\Record;
+use pocketmine\math\Vector3;
+use pocketmine\nbt\tag\CompoundTag;
+use pocketmine\world\sound\RecordSound;
+use pocketmine\world\sound\RecordStopSound;
+
+class Jukebox extends Spawnable{
+	public const TAG_RECORD = "record";
```

Comment:
This should be `private`. Anything outside of this class has no business using it.
Please also comment next to it what the expected type of tag is.

Reply:
The wiki says that the tag is `RecordItem`. Please check that PC jukeboxes can be decoded correctly.

Reply:
Sorry, just copied other tile format most are public with no type.